definitions/install.yaml

```diff
@@ -495,13 +495,27 @@ metadata:
 apiVersion: rbac.authorization.k8s.io/v1beta1
 kind: ClusterRole
 metadata:
-  name: kyverno:policyviolations
+  name: kyverno:view-policyviolations
+  labels:
+    rbac.authorization.k8s.io/aggregate-to-view: "true"
```

Comment:
@F1ko - should we also add `rbac.authorization.k8s.io/aggregate-to-edit: "true"` to allow view access for the edit role?

Reply:
@JimBugwadia 
There is no need for that.
By default, clusterrole `view` does aggregate to `edit` anyways, which means that this way of doing it is enough.

Reply:
@F1ko - ok, was wondering about that. Thanks for clarifying!